Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: consul
-version: 0.9.0-beta
+version: 0.9.1-beta
```

Comment:
We'll make this change when we do a release. Please remove it for now because it causes issues with merging.